vendor/licenses/git_submodule/urscript-extension.dep.yml

```diff
@@ -1,22 +1,26 @@
 ---
-name: typst-grammar
-version: 180d59e6f4e09941d778b5fa6fc48d52211294a1
+name: urscript-extension
```

Comment:
You need to merge main into your fork. Your PR should not be renaming and changing the `typst-grammar.dep.yml` file; it should only be adding new file.

Reply:
I am not sure how that happened. I hope it is in order now🤞

Reply:
Yup. Looking good now.